Userland/Libraries/LibGUI/ItemListModel.h

```diff
@@ -63,7 +63,7 @@ class ItemListModel : public Model {
         return "Data";
     }
 
-    virtual Variant data(const ModelIndex& index, ModelRole role) const override
+    virtual Variant data(const ModelIndex& index, ModelRole role = ModelRole::Display) const override
```

Comment:
One note, default parameters on virtual methods are usually not useful, where is this needed?

Reply:
It's used in `if (!string_matches(data(index).as_string(), searching, flags))`.
Should I change it to explicitly say that it wants `ModelRole::Display`?

Reply:
Yeah, let's pass that explicitly.